tools/harness-simulation/README.md

```diff
@@ -0,0 +1,28 @@
+# Test Harness on Simulation Environment Setup
+
+THCI (Thread Host Controller Interface) is an implementation of the Python abstract class template `IThci`, which is used by the Thread Test Harness Software to control OpenThread-based reference devices according to each test scenario.
+
+SI (Sniffer Interface) is an implementation of the sniffer abstract class template `ISniffer`, which is used by the Thread Test Harness Software to sniff all packets sent by devices.
+
+Both OpenThread simulation and sniffer simulation are required to run on a POSIX environment. However, Harness has to be run on Windows, which is a non-POSIX environment. So both two systems are needed, and their setup procedures in detail are listed in the following chapters. Either two machines or one machine running two (sub)systems (e.g., VM, WSL) is feasible.
+
+Platform developers should modify the THCI implementation and/or the SI implementation directly to match their platform (for example, the path of the OpenThread repository).
+
+## POSIX Environment Setup
+
+1. Build OpenThread to generate standalone OpenThread simulation `ot-cli-ftd`. For example, run `script/cmake-build simulation` in the top directory of OpenThread. Then `ot-cli-ftd` is built in the directory `build/simulation/examples/apps/cli/`.
```

Comment:
Can you make the command a code block so it's more clear the command that is being run?  It can easily get lost within a paragraph.  See the example in the next comment